.github/workflows/kotlin.yml

```diff
@@ -34,6 +34,17 @@ jobs:
         with:
           java-version: 1.8
 
+      - name: setup environment secrets
+        shell: bash
+        env: # pull in the secrets and environment variables
+          PORCHBOX_GOOGLE_SERVICES: $${{ secrets.GoogleServices }}
```

Comment:
Can an admin add these secrets for me if we are OK with this?  @zach-klippenstein @rjrjr